src/server/main.py

```diff
@@ -1,7 +1,32 @@
+import database_provider
+
 from flask import Flask
 
 app = Flask(__name__)
```

Comment:
Do we need to store any state during server runtime? I realize that Flask devs generally recommend creating a setup on a module level, but in case we need to save some data, it would be great to wrap the server logic in a class.

Reply:
Pewnie nie, ale będzie wymagane połączenie z bazą danych